usr/lib/config/configuration.c

```diff
@@ -655,3 +661,22 @@ confignode_allocnumpairlistdumpable(char *key, char* end,
     }
     return res;
 }
+
+struct ConfigBareStringConstNode *
+confignode_allocbarestringconstdumpable(char *key, int line, char *comment)
+{
```

Comment:
When looking at the existing config-related code in OCK, parameters like "key" are usually consts. Could "char *key" be "const char *key" here as well? Maybe even "comment". Also other places below.

Reply:
Well, these things could probably be const, but they are not on all the other config structures and functions, so I guess I will leave it as is for now. 
Maybe we can do this for all places at a later time.